src/components/autocomplete/js/autocompleteController.js

```diff
@@ -359,8 +359,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
    * Handles input blur event, determines if the dropdown should hide.
    */
   function blur () {
-    hasFocus = false;
```

Comment:
@robertmesserle Can you think of any reason why the `hasFocus` was outside of the `if` statement? All tests pass after moving it and my manual testing didn't show any differences except that it fixed my bug.